setup/www/resources/config/nodejs.org

```diff
@@ -45,6 +45,8 @@ server {
     }
 
     location / {
+        try_files $uri $uri/ @english_fallback;
```

Comment:
Right, so what I'm trying to suggest is a way to get the [localized 404 page](https://github.com/nodejs/nodejs.org/blob/master/locale/ko/404.md)s. The rewrite in `@english_fallback` prevents that I think, but at the same time, it doesn't currently work either and I'm not so sure why.

Then again if the localized 404 doesn't exist, all bets are off but I don't think that is priority and 404 pages are super easy to make.